.actrc

```diff
@@ -0,0 +1,2 @@
+--artifact-server-path /tmp/act-artifacts
```

Comment:
I understand this helps configure `act` for this repo, though can we please add comments to that effect?

Reply:
Double checking support